package.json

```diff
@@ -16,6 +16,9 @@
   "pnpm": {
     "overrides": {
       "hardhat>@nomicfoundation/edr": "workspace:*"
+    },
+    "patchedDependencies": {
+      "hardhat@2.22.6": "patches/hardhat@2.22.6.patch"
```

Comment:
Ah, I hadn't understood that these patches are applied automatically on installation, cool!